### PR TITLE
Skip already handled GitHub events

### PR DESCRIPTION
**Pull request by Half-Shot**
This is just a simple check to ensure we don't handle multiple events sent by GitHub by looking at the GUID provided. This is already pretty rare, but we've seen this happen rarely in the wild.

GitLab/JIRA don't support this as they don't provide a guid :s

### CODE REVIEW

File: src/Webhooks.ts [hunk 1]
```diff
@@ -139,15 +141,20 @@ export class Webhooks extends EventEmitter {
         try {
             let eventName: string|null = null;
             const body = req.body;
-            if (req.headers['x-hub-signature']) {
```
Comment (tadzik) [inline src/Webhooks.ts:142]: We check for the a different header now, but Github [always sends both](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads).

File: src/Webhooks.ts [hunk 2]
```diff
@@ -37,8 +38,9 @@ export interface NotificationsDisableEvent {
 export class Webhooks extends EventEmitter {
     
     public readonly expressRouter = Router();
-    private queue: MessageQueue;
-    private ghWebhooks?: OctokitWebhooks;
+    private readonly queue: MessageQueue;
+    private readonly ghWebhooks?: OctokitWebhooks;
+    private readonly guids = new QuickLRU<string, void>({ maxAge: 5000, maxSize: 100 });
```
Comment (tadzik) [inline src/Webhooks.ts:43]: Maybe `handledGuids` (, to make it more obvious what we keep them for.

**Review by tadzik — CHANGES_REQUESTED** [review]

File: src/Webhooks.ts (hunk 1 above)
Comment (Half-Shot) [reply src/Webhooks.ts:142]: I think this is okay, the delivery header is always sent?

Comment (tadzik) [reply src/Webhooks.ts:142]: Yeah, that's what I meant :)

**Review by tadzik — APPROVED** [review]

**Pull request merged**